test/Microsoft.Azure.Devices.Edge.Test.Common/IPlatform.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Security.Cryptography.X509Certificates;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Azure.Devices.Edge.Test.Common.Certs;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public interface IPlatform
```

Comment:
```suggestion
    public interface IOsPlatform
```

Reply:
I don't like it--it's redundant (the terms "OS" and "platform" are often used interchangeably). But I see that .NET uses it in their `RuntimeInformation` class, so I guess I can live with it. Updated.